src/assets/defaults.ts

```diff
@@ -901,3 +901,11 @@ export const defaultSensorDataloggerProfile: OverlayGrid = {
 export const defaultDisplayUnitPreferences = {
   distance: DistanceDisplayUnit.Meters,
 }
+
+export const defaultAutoRecordingOptions = {
+  autoRecordOption: undefined,
```

Comment:
`autoRecordMode` seems like a better choice of name.

Whenever you have an object with a name and a property with the same name or very similar (e.g.: AutoRecordOptions > autoRecordOption), it usually means there's an architectural problem (not the case here) or a naming problem.

Reply:
I think you got it wrong. The object should stay as `autoRecordOptions` (as there are several options inside the object) and the property is the one to be renamed to `autoRecordMode` (as it has modes, like `all`, `none` and `selected`).